extensions/wikia/StaffSig/StaffSig.php

```diff
@@ -21,20 +21,24 @@ function wfSigSetup(&$parser) {
 
 function wfMakeStaffSignature( $contents, $attributes, $parser ) {
 	$title = GlobalTitle::newFromText('Staff', 4 /*project*/, 177);
-	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a member of Wikia Staff" );
+	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a member of Fandom Staff" );
 }
 
 function wfMakeHelperSignature( $contents, $attributes, $parser ) {
 	$title = GlobalTitle::newFromText('Helper_Group', 12 /*help*/, 177);
-	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a Wikia Helper" );
+	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a Fandom Helper" );
 }
 
 function wfMakeSignatureCommon($href, $title, $iurl=null) {
 	global $wgExtensionsPath, $wgBlankImgUrl;
 
 	if( empty($iurl) ) {
-		$iurl = $wgExtensionsPath . '/wikia/StaffSig/images/WikiaStaff.png';
+		$iurl = wfGetSignatureUrl();
 	}
 
 	return '<a href="'. $href .'" title="'. $title . '" class="staffSigLink"><img src="'. $wgBlankImgUrl .'" style="background-image: url('. $iurl .')" alt="@Wikia" class="staffSig" width="41" height="12" /></a>';
 }
+
+function wfGetSignatureUrl() {
+	return $wgExtensionsPath . '/wikia/DesignSystem/bower_components/design-system/assets/company/logo-fandom.svg';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgExtensionsPath" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)